docs/user/site.rst

```diff
@@ -181,6 +181,20 @@ wifi24 \: optional
 wifi5 \: optional
     Same as `wifi24` but for the 5Ghz radio.
 
+    Additionally a range of channels that are safe to use outsides on the 5 GHz band can
```

Comment:
As I think this should be a separate package, you should write something like "with the package xxx".

Reply:
As this feature modifies the logic of wifi interface configuration, moving this into a separate package will make the implementation significantly more complex. If we are actually planning to modularize the interface config (which I don't see at this point), this PR is not the place to discuss it.